templates/docker-compose.yml.j2

```diff
@@ -71,6 +72,17 @@ services:
 {% if go_waku_rendezvous_server_enabled %}
       --rendezvous-server
       --rendezvous-db-path=/data/rendezvous
+{% endif %}
+{% if go_waku_dns_disc_enabled %}
+      --dns-discovery=true
```

Comment:
I think I'd rather have the toggle flag outside of the if with the boolean formatted using `to_json`. Same below.